### ISSUE TITLE
Remote branch error when creating PR

### ISSUE THREAD

**Issue by vitorf7** [pwntester/octo.nvim#437]
## Issue Description

Type: *bug report* 

### Describe what happened (or what feature you want)
Recently I have been getting a weird error where I get weird characters appearing when the `Remote Branch` step of the PR creation pops up. Whether I simply open `:Octo` or do `:Octo pr create` I get the same error as below:
![Screenshot 2023-10-17 at 18 20 16](https://github.com/pwntester/octo.nvim/assets/469102/64bbc164-44b0-4d3e-b592-41c07ce6c6e5)

### Describe what you expected to happen
I expected the steps to ask what the remote branch, etc to work as it used to a few weeks ago

### How to reproduce it (as minimally and precisely as possible)

1. Open neovim and change a file. 
2. Open `lazygit` and push local changes to a new remote branch
3. Run `:Octo pr create`

### Tell us your environment
MacOS 14.0
Alacritty 0.12.3
Neovim 
```shell
NVIM v0.10.0-dev-1080+g676e1d490
Build type: RelWithDebInfo
LuaJIT 2.1.1694316387
```
gh 2.36.0 (2023-10-03)
octo `master e933611c74cb735c00d0db8ca94533bec1444bf0`

### Anything else we need to know?
- `$GITHUB_TOKEN` is set to a new personal token I have created
- I did `gh auth logout` and then `gh auth login` and I followed the steps to use `ssh` and pasted in the same personal token that I had in my `$GITHUB_TOKEN`
- Doing `gh pr create` works fine

**Comment by vitorf7** [comment]
**Update:**

This happened because I had no idea how to work with my new shell (`fish`).
I had `neofetch` at the bottom of my `config.fish` but I just learned that this means it runs whether it is an interactive or non interactive shell.
For other's benefits, the way I got this working was setting a variable in fish called `fish_greeting`. I had to do the following:
```fish
function fish_greeting
    neofetch
end

funcsave fish_greeting
```
